### PR TITLE
Add Slider

### PR DESCRIPTION
**Pull request by quadreex**

### CODE REVIEW

**Comment by extronics** [comment]
Small layout glitch: the "required" asterisk should be on the same line as the label:

image

File: src/components/Form/__tests__/Slider.spec.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import { mount } from 'enzyme';
+
+import { getUniqueHash } from '../../../utils';
```
Comment (extronics) [inline src/components/Form/__tests__/Slider.spec.js]: Please remove, yields an linter error

**Comment by quadreex** [comment]
@extronics @fspoettel Does it really make sense to have `required` flag for this component, as we always have value for it?
Maybe I'd better remove this flag?

**Comment by extronics** [comment]
@quadreex you're right. Also, the salesforce design system doesn't seem to provide a `required` state for the slider at all.

**Pull request merged**